dotCMS/src/main/java/com/dotmarketing/services/ContainerServices.java

```diff
@@ -284,21 +285,27 @@ public static InputStream buildVelocity(Container container, Identifier identifi
             sb.append(container.getCode());
         }
 
+        OutputStreamWriter out = null;
+        BufferedOutputStream tmpOut = null;
+
         try {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Change this "try" to a try-with-resources. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2093)

Reply:
not valid